sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -267,7 +265,7 @@ case class FileSourceScanExec(
     val metadata =
       Map(
         "Format" -> relation.fileFormat.toString,
-        "ReadSchema" -> outputSchema.catalogString,
+        "requiredSchema" -> requiredSchema.catalogString,
```

Comment:
This is also for display in `SparkPlanInfo`? Keep the original name `ReadSchema`?